src/OpenSpaceToolkit/Astrodynamics/EventCondition/Conjunctive.cpp

```diff
@@ -0,0 +1,38 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Astrodynamics/EventCondition/Conjunctive.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace eventcondition
+{
+Conjunctive::Conjunctive(const Array<Shared<EventCondition>>& eventConditions)
+    : LogicalConnective("Conjunctive Condition", eventConditions)
+{
+}
+
+Conjunctive::~Conjunctive() {}
+
+bool Conjunctive::isSatisfied(
+    const VectorXd& currentStateVector,
+    const Real& currentTime,
+    const VectorXd& previousStateVector,
+    const Real& previousTime
+) const
+{
+    for (const Shared<EventCondition>& eventCondition : eventConditions_)
```

Comment:
Not that important, but `std::all_of` could be used here and has closer semantics to the AND nature of this class. Could be slightly better readability:
```
return std::all_of(
  eventConditions_.begin(),
  eventConditions_.end(),
  [&](EventCondition& condition){return condition->isSatisfied(currentStateVector, currentTime, previousStateVector, previousTime)}
);
```
Totally a style thing, so your call. Feel free to close this comment without responding.